lib/rspec/matchers/built_in/include.rb

```diff
@@ -87,6 +89,7 @@ def excluded_from_actual
           return [] unless @actual.respond_to?(:include?)
 
           expecteds.inject([]) do |memo, expected_item|
+            expected_item = expected_item.to_hash if expected_item.respond_to?(:to_hash)
```

Comment:
Nice PR!

The discussion in #978 concluded that the `expecteds` did not need to be converted to a hash. Is this necessary?

Reply:
Aha, I had missed that conversation. I agree with the conclusion, so I have removed this conversion now.